lib/xls_function/evaluators/functions/switch.rb

```diff
@@ -0,0 +1,25 @@
+module XlsFunction
+  module Evaluators
+    module Functions
+      class Switch < ::XlsFunction::Evaluators::FunctionEvaluator
+        function_as :switch
+
+        def eval_arglist
+          # Skip common argument evaluation and assignment for short-circuit
+        end
+
+        def eval
+          condition = arg_list.first.evaluate(context)
+
+          arg_list[1..-1].each_slice(2) do |expr, value|
+            return value&.evaluate(context) if expr&.evaluate(context) == condition
+
+            return expr&.evaluate(context) if value&.evaluate(context).nil?
```

Comment:
`expr&.evaluate(context)` runs twice. Plz fix it.

Reply:
I fixed in the commit below.
[b066e5d](https://github.com/moneyforward/xls_function/pull/16/commits/b066e5d4e257fb69bca6be69fd93a793b032105d)